fastembed/text/onnx_embedding.py

```diff
@@ -14,7 +14,7 @@
         "description": "Base English model",
         "size_in_GB": 0.42,
         "sources": {
-            "url": "https://storage.googleapis.com/qdrant-fastembed/fast-bge-base-en.tar.gz",
+            "hf": "aident-ai/bge-base-en-onnx",
```

Comment:
you can use `yashvardhan7/bge-base-en-onnx` here

Reply:
Thanks Yash. We're probably will proceed with #201 instead.

Reply:
Committed your source. Just in case.